vireo/templates/pipeline_review.html

```
@@ -1412,36 +1433,32 @@ <h3 id="grmTitle">Review Burst Group</h3>
       </div>
     </div>
     <div class="grm-loupe" id="grmLoupe">
-      <div class="grm-loupe-img" id="grmLoupeImg" onmousemove="grmLoupeMove(event)" onmouseleave="grmLoupeReset()" onclick="grmLoupeToggleLock(event)" onwheel="grmLoupeZoom(event)">
+      <div class="grm-loupe-img" id="grmLoupeImg" onmousedown="grmLoupePointerDown(event)" onmousemove="grmLoupeMove(event)" onmouseup="grmLoupePointerUp(event)" onmouseleave="grmLoupeReset()" onclick="grmLoupeToggleLock(event)" onwheel="grmLoupeZoom(event)">
         <img id="grmLoupePhoto" src="" alt="">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)